### ISSUE TITLE
Solid pulmonary nodule

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6634]
**Preferred term label:**Solid pulmonary nodule

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Pulmonary nodule (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik  Adding the following definition, please reopen this issue if it is wrong

def A type of pulmonary nodule whose density  obscures the underlying parenchyma and tghus has a "solid" appearance. 
PMID:29143191
